themes/bootstrap3/js/vendor/iframemanager.js

```diff
@@ -0,0 +1,17 @@
+/*
+ iframemanager v1.0
```

Comment:
Should we be loading this via NPM as well?

Reply:
Yes, but unfortunately it's not available there.

Reply:
Too bad! Maybe we could add a download task for it in Phing, then... though it's annoying to have to manage versions of things in multiple places. (I guess there's also the question of whether/how well this will be maintained, since as of right now it has only one release).

Reply:
Making it reliable and somehow integrate with npm's or composer's 'outdated' check might be bit of a burden. The library hasn't had any frequent updates either, so while this is not optimal, I feel like the additional work is not worth the trouble.

Reply:
From my perspective, the real question is whether this code has a future. If it does not, are we just setting ourselves up for some kind of emergency rewrite/replacement down the road? How heavily do we rely on this, and is there anything we can/should do now to protect ourselves against future problems?

Reply:
If it's easy to separate the iframemanager piece, I'd even be open to moving that to a separate PR -- then we could merge the base behavior right away, and leave the iframemanager PR open a little longer to see how things develop and consider alternate approaches if necessary.

Reply:
It's now gone from here, and I replaced the sample template on the wiki page with one that doesn't need iframemanager.

Reply:
One approach that would remove the need for iframemanager or such but would make it easier to embed videos is to add a view helper for that. It would then render the template code like in the [YouTube example](https://vufind.org/wiki/configuration:cookie_consent#youtube).

Reply:
Yes, definitely worth considering! I guess to a large extent it may come down to community needs and whatever makes your local code easier to maintain. I'm open to a variety of possibilities; I just wanted to be careful about setting a bad precedent for accepting questionable dependencies. :-)